### PR TITLE
fix: make feed valid, sort by publishing date (with fallback), reformat and clean up

### PR DESCRIPTION
**Pull request by j9t**
Closes DX-112

### CODE REVIEW

File: feed.xml [hunk 1]
```diff
@@ -2,31 +2,24 @@
...
+    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+    <generator>Jekyll v{{ jekyll.version }}</generator>
+    {% assign pages = site.pages | sort: "date_published" | reverse %}
+    {%- for page in pages limit:25 -%}
+    <item>
+      <title>{{ page.pagename | xml_escape }}</title>
+      <link>{{ page.url | prepend: site.url }}</link>
+      <guid isPermaLink="true">{{ page.url | prepend: site.url }}</guid>
+      <description><![CDATA[{{ page.content | xml_escape }}]]></description>
+      <!-- <pubDate>{{ page.date_published | default: "2017/05/17" | date_to_rfc822 }}</pubDate> -->
```
[the remark this answers is not in the document]
Comment (j9t) [reply feed.xml:21]: It’s optional by the way, which is why I commented it; I left it in, however, to make debugging a bit easier, at least for now.

File: feed.xml [hunk 2]
```diff
@@ -2,31 +2,24 @@
...
-</channel>
-</rss>
+  <channel>
+    <title>{{ site.site_title | xml_escape }}</title>
+    <link>{{ site.url }}</link>
+    <atom:link href="{{ "/feed.xml" | prepend: site.url }}" rel="self" type="application/rss+xml" />
+    <description>{{ site.description | xml_escape }}</description>
+    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+    <generator>Jekyll v{{ jekyll.version }}</generator>
+    {% assign pages = site.pages | sort: "date_published" | reverse %}
```
Comment (j9t) [inline feed.xml:14]: @itay1313 FYI: This line governs sorting by pub date, which needs to be in reverse.

**Review by j9t — COMMENTED** [review]
(Left comments to explain changes.)

File: feed.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (itay1313) [reply feed.xml:21]: nice, thanks for taking this DX-112, let's merge this and I will check the validation. thanks

**Review by itay1313 — APPROVED** [review]

**Comment by j9t** [comment]
Thanks for the review!

**Pull request merged**